Server/mods/deathmatch/logic/CPed.h

```diff
@@ -103,6 +103,42 @@ enum eBone
     BONE_RIGHTFOOT
 };
 
+struct SPlayerAnimData
+{
+    std::string blockName;
+    std::string animName;
+    int         time;
+    bool        loop;
+    bool        updatePosition;
+    bool        interruptable;
+    bool        freezeLastFrame;
+    int         blendTime;
+    bool        taskToBeRestoredOnAnimEnd;
+
+    SPlayerAnimData()
+        : blockName(""),
+          animName(""),
+          time(-1),
```

Comment:
Please, use default member initialization instead of the member initializer list. I mean `bool loop{true};`.